__tests__/tests.test.js

```diff
@@ -54,7 +56,7 @@ describe("/api", () => {
       });
   });
 });
-describe.only("/api/topics", () => {
+describe("/api/topics", () => {
```

Comment:
you've got /api/articles/:article_id within your describe block for /api/topics :) Make sure this gets a new describe block!